### ISSUE TITLE
Make Zeroconf browser more like the Chooser

### ISSUE THREAD

**Issue by probonopd** [helloSystem/Utilities#1]
Currently we have

![image](https://user-images.githubusercontent.com/2480569/100012911-50fce600-2dd4-11eb-88f2-61637b836416.png)

It can become hard to find stuff real quick.

Maybe we should make it a two column browser, with the left column showing the available services on the network and the right column the individual instances of those services?

Similar to
https://en.wikipedia.org/wiki/Chooser_(Mac_OS)

![image](https://user-images.githubusercontent.com/2480569/100022315-3b42ed00-2de3-11eb-85b7-43e130106364.png)

**Comment by panosl** [comment]
> Very cool @panosl - are these mockups or actually written in PyQt5?
> The current code is at https://github.com/helloSystem/Utilities/blob/master/Zeroconf.app/Zeroconf.

Nope, sketch mockups. It's easier if the usability is discussed/decided upon first, before the actual code is written (although from what I see PyQt looks quite simple/straightforward).

**Comment by probonopd** [comment]
Also see https://github.com/helloSystem/Menu/issues/19 - how do you like that idea? Maybe even cooler, because no extra chooser application would be needed.
